src/frontend/src/features/layouts/components/mailbox-selector/_index.scss

```
@@ -97,3 +97,44 @@
+        line-height: 1;
+
+        &:before {
+            content: '';
+            display: inline-block;
+            --size: 0.5rem;
+            flex-shrink: 0;
+            width: var(--size);
+            height: var(--size);
+            border-radius: 50%;
+            box-sizing: border-box;
+            background-color: transparent;
```

Stylelint (17.14.0)

[error] Duplicate property "background-color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)